README.md

```diff
@@ -9,23 +9,159 @@ An engineer working for a [CSE](./CSE.md) project...
 
 This is the playbook. You are invited to contribute via standard git pull requests to GitHub.
```

Comment:
This is out of scope, but we really should improve this language. There's ofc no such thing as a "git pull request". As it stands, this reads as somewhat embarrassing, so we ought to fix it quickly.

Reply:
updated language